lib/burials/notification_email.rb

```diff
@@ -13,6 +14,7 @@ def initialize(saved_claim_id)
 
     private
 
+    # @see VeteranFacingServices::NotificationEmail::SavedClaim#claim_class
     def claim_class
       SavedClaim::Burial
```

Comment:
@wayne-weibel The claim_class will change to `Burials::SavedClaim` when the module is fully in use FYI